src/components/jar_details/UtxoList.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { useMemo } from 'react'
+import * as rb from 'react-bootstrap'
+import classnames from 'classnames'
+import { useTranslation } from 'react-i18next'
+import { Table, Header, HeaderRow, HeaderCell, Body, Row, Cell } from '@table-library/react-table-library/table'
+import { useRowSelect, HeaderCellSelect, CellSelect, SelectTypes } from '@table-library/react-table-library/select'
+import { useSort, HeaderCellSort, SortToggleType } from '@table-library/react-table-library/sort'
+import * as TableTypes from '@table-library/react-table-library/types/table'
+import { useTheme } from '@table-library/react-table-library/theme'
+import { useSettings } from '../../context/SettingsContext'
+import { Utxo, WalletInfo } from '../../context/WalletContext'
+import * as fb from '../fb/utils'
+import Sprite from '../Sprite'
+import Balance from '../Balance'
+import styles from './UtxoList.module.css'
+
+const ADDRESS_STATUS_COLORS: { [key: string]: string } = {
+  new: 'normal',
+  used: 'normal',
+  reused: 'danger',
+  'cj-out': 'success',
+  'change-out': 'warning',
+  'non-cj-change': 'normal',
+  deposit: 'normal',
+}
+
+const SORT_KEYS = {
+  frozenOrLocked: 'FROZEN_OR_LOCKED',
+  value: 'VALUE',
+  confirmations: 'CONFIRMATIONS',
+}
+
+const TABLE_THEME = {
+  Table: `
+    --data-table-library_grid-template-columns: 2rem 3.5rem 2fr 4fr 6rem 3fr 1fr;
+    font-size: 0.9rem;
+  `,
+  BaseCell: `
+    &:nth-of-type(1) {
+      text-align: center;
+    }
+  `,
```

Comment:
```suggestion
  BaseCell: `
    &:nth-of-type(1) {
      text-align: center;
    }
    &:nth-of-type(3) button {
      display: flex;
      justify-content: end;
    }
    &:nth-of-type(5) button {
      display: flex;
      justify-content: center;
    }
  `,
```

This change aligns the column header accordingly. Just a thought. Yes/no?

<img src="https://user-images.githubusercontent.com/3358649/182370869-2fd5e9ef-7061-4986-9c56-5519d47b2266.png" width=400 />

Reply:
Oh yeah good point. We can remove the code below as well then. `BaseCell` will also apply to `Cell`.